app/scripts/constants.js

```diff
@@ -377,5 +377,7 @@ window.OPENSHIFT_CONSTANTS = {
         }
       ]
     }
-  ]
+  ],
+  // 'openshift' should always be included
+  CREATE_FROM_URL_WHITELIST: ['openshift']
```

Comment:
I'd move this above `PROJET_NAVIGATION` because it's a bit easy to miss below the two long constants.

Suggest adding more comments describing what it is. I'd like the file to be as self-documenting as possible since we expect admins to customize these values.